### PR TITLE
Use new attribution table

### PR DESCRIPTION
**Pull request by fbertsch**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql [hunk 1]
```diff
@@ -53,28 +53,15 @@ first_seen AS (
...
-    ARRAY_AGG(metrics.string.first_session_campaign ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_campaign,
-    ARRAY_AGG(metrics.string.first_session_creative ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_creative,
-    MIN(DATE(submission_timestamp)) AS first_session_date
+    client_id,
+    adjust_network,
+    adjust_adgroup,
```
Comment (lucia-vargas-a) [inline sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql:58]:
```suggestion
    adjust_ad_group AS adjust_adgroup,
```

Comment (lucia-vargas-a) [reply sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql:58]: Or even better, could we update the column name to use the same naming in both tables? Would be easier to find & query data.

**Review by lucia-vargas-a — APPROVED** [review]
+ Requires a column name update.

File: sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/init.sql [hunk 2]
```diff
@@ -63,28 +63,15 @@ CLUSTER BY
...
-        ARRAY_AGG(metrics.string.first_session_campaign ORDER BY submission_timestamp)[
-          SAFE_OFFSET(0)
-        ] AS adjust_campaign,
-        ARRAY_AGG(metrics.string.first_session_creative ORDER BY submission_timestamp)[
-          SAFE_OFFSET(0)
-        ] AS adjust_creative,
-        MIN(DATE(submission_timestamp)) AS first_session_date
+        client_id,
+        adjust_network,
+        adjust_adgroup,
```
Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/init.sql:68]:
```suggestion
        adjust_ad_group AS adjust_adgroup,
```

**Pull request merged**